editor/editor_node.cpp

```diff
@@ -2689,6 +2689,15 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 			}
 			[[fallthrough]];
 		}
+
+		case FILE_SAVE_SCENE_SILENTLY: {
+			Node *scene = editor_data.get_edited_scene_root();
+			if (scene && !scene->get_scene_file_path().is_empty() && DirAccess::exists(scene->get_scene_file_path().get_base_dir())) {
+				_save_scene(scene->get_scene_file_path());
+				save_editor_layout_delayed();
+			}
+		} break;
```

Comment:
This will break the save logic, see how the previous case has a `[[fallthrough]]` attribute and no `break;` - so it's meant to fall through to the `FILE_SAVE_AS_SCENE` case, which won't happen anymore here.

Reply:
Missed it, thank you! 
I believe, it should be fixed now.